guidelines/index.html

```diff
@@ -2922,9 +2922,9 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </dd>
 
             
-            <dt class="new"><dfn data-lt="bounding boxes|minimum bounding box" data-plurals="bounding box" id="dfn-bounding-boxes" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">minimum bounding box</dfn></dt>
+            <dt class="new"><dfn data-lt="bounding boxes|minimum bounding box" data-plurals="bounding box" id="dfn-bounding-boxes" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">最小境界ボックス (minimum bounding box)</dfn></dt>
 <dd class="new"><p class="change">[New]</p>	
-   <p>the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie. For components which wrap onto multiple lines as part of a sentence or block of text (such as hypertext links), the bounding box is based on how the component would appear on a single line.</p>
+   <p>ある図形のすべての点が入る、水平軸に沿ってもっとも小さく囲まれた矩形。文あるいはテキストブロックの一部として、複数行に折り返されるコンポーネント (ハイパーテキストのリンクなど) の場合、境界ボックスは、そのコンポーネントが 1 行であったときにどう表示されるかに基づく。</p>
```

Comment:
状況としてはこういうことなのかなと思います。
<https://imagingsolution.net/program/python/opencv-python/contour_bounding_rect/>

lieを単純に「置いてある」ですとちょっと訳として微妙ですか…「入る」ととりあえずしておきましょうか…
aligned to the horizontal axisについては、水平線に沿って配置されると取れますがこれは結局「水平線と平行」ということかな思います。
smallest enclosing rectangleは後述のbounding boxで「境界ボックス」としているので、「境界」としますか。

2文目はもにょっと語順を入れ替えたりしてみました。

```suggestion
   <p>ある図形の全ての点が入る、水平軸と平行な最小境界矩形。文又はテキストブロック  (ハイパーテキストのリンクなど) の一部として、複数行に折り返されるコンポーネントの場合、境界ボックスは、そのコンポーネントが単一行でどのように表示されるかに基づく。</p>
```